lips/lip-10.md

```diff
@@ -0,0 +1,221 @@
+* Sender Libra ID must have a Libra ID domain that belongs to Sender VASP. If this is not the case, the Receiver VASP should return the `400` error code 
+* Receiver VASP may optionally choose how much information to share based on the sender Libra ID. If Receiver VASP decides to not share data, it should return an empty `User Object` with only `libra_id` field set
+* If user is not found, the VASP should return error code `400`  
+
+## Pay endpoint
+
+```
+POST https://<hostname>:<port>/pay.<protocol_version>/pay
+```
+
+**Pay endpoint** is used when a user is willing to make a money transfer.
+
+**Request:**
+```json
+{
+   "sender": "<User Object>",
+   "receiver_id": "<Receiver_Libra_ID>",
+   "amount": "<amount>",
+   "currency": "<currency>"
+}
+```
+
+| Field 	   | Type 	     | Required? 	| Description 	           |
+|-------	   |------	     |-----------	|-------------	           |
+|sender        |[User Object](https://github.com/libra/lip/blob/master/lips/lip-10.md#user-object)  | Y            | Information about the sender |
+|receiver_id   |str          | Y            | Libra ID of the receiver |
+|amount        |integer      | Y            | Amount of coins to transfer |
+|currency      |str          | Y            | Name of the coin |
+
+**Response:**
+```json
+{
+   "receiver_address": "<Receiver VASP on-chain account address>",
+   "reference_id": "<Payment_Reference_ID>",
+   "receiver": "<User Object>"
+}
+```
+
+| Field 	      | Type       | Required? 	| Description 	           |
+|-------	      |------      |-----------	|-------------	           |
+|receiver_address |str         | Y            | On-chain account address for receiving the payment. This address must be encoded as a [bench 32 account identifier](https://github.com/libra/lip/blob/master/lips/lip-5.md#account-identifiers) **without** subaddress |
+|reference_id     |str         | Y            | Reference ID to be included into payment metadata |
+|receiver         |[User Object](https://github.com/libra/lip/blob/master/lips/lip-10.md#user-object) | Y            | Information about the receiver |
+
+* Pay endpoint is used to exchange details of the payment such as information about Sender and Receiver off the chain, and negotiate `Payment_Reference_ID`
+
+* Once the sender learns Payment_Reference_ID, it will submit an on-chain transaction including the `Payment_Reference_ID` as a metadata. Receiver will use the `Payment_Reference_ID` in the transaction to link the on-chain transaction to the payment.
+
+* Sender will submit a transaction to the `receiver_address` on-chain account in the response. This account address can be either Parent VASP, or any of Child VASP accounts of the receiver VASP. This is needed to allow the receiver VASP to manage load between different on-chain accounts.
+
+* `Payment_Reference_ID` must be unique for each payment, it must not be reused in any way to avoid leaking privacy information on-chain.
```

Comment:
who enforce this? in order for this to be meaningful we would need this to be enforced by at least one VASP, they probably won't unless we have a certification process, or the blockchain.